rmgpy/rmg/react.py

```diff
@@ -124,14 +88,11 @@ def react_species(species_tuple, only_families=None):
     return reactions
 
 
-def react_all(core_spc_list, numOldCoreSpecies, unimolecularReact, bimolecularReact, trimolecularReact=None):
+def react_all(core_spc_list, numOldCoreSpecies, unimolecularReact, bimolecularReact, procnum, trimolecularReact=None):
```

Comment:
Maybe add a default value of 1 for procnum?

Reply:
done